changelog.md

```diff
@@ -2,6 +2,10 @@
 
 ## v1
 
+### v1.1.3
```

Comment:
v1.1.3 has already been added in https://github.com/Financial-Times/x-dash/pull/483, so you'll need to rebase.

In any case, the [Release Guidelines](https://github.com/Financial-Times/x-dash/blob/master/docs/components/release-guidelines.md#user-content-releasingversioning) say:
> [before updating `changelog.md`] it assumes that all the code you wish to release is now on the master or main branch

so this change should be added in a separate branch and merged to master with a commit message of "Version x.x.x.".

Having said all that, I will raise with Platforms team at our next huddle with the full team of devs (Tue 30 Jun 2020) whether we want to keep `changelog.md` (given it essentially repeats what the GitHub GUI Releases tab already displays).

Reply:
@andygout  I've rebased so not sure why this is showing as a change on this PR at all.